experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1502,155 +1503,6 @@ def test_serializer_no_fml_validation_on_non_mobile(self):
         self.assertTrue(serializer.is_valid())
         self.mock_fml_errors.assert_not_called()
 
-    def test_serializer_fml_invalid_reference_branch_value(self):
```

Comment:
These are no longer needed in the `TestNimbusReviewSerializerSingleFeature` section, since everything should be versioned now. These tests are covered below in `VersionedFeatureValidationTests` section